qa/sc_evm_estimateGas.py

```diff
@@ -90,7 +90,7 @@ def run_test(self):
         to = '0x0000000000000000000011111111111111111111'
```

Comment:
After merge with dev_evm the address of this smart contract has changed from `9950a60f` into `4267ec5e`

Reply:
pls see commit 064e43f3f

Reply:
fixed